src/components/selectInput/selectInput.css

```diff
@@ -0,0 +1,83 @@
+*{
+    font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+}
+
+.slctInputLabel{
+    font-size: 14px;
+    font-weight: bold;
+    color:gray;
+}
+
+.slctInputContainer{
+    display: flex;
+    flex-direction: column;
+    width: auto;
+    min-width: 100px;
```

Comment:
cursor pointer needs to be added as well on each item , user interaction is importanr